CHANGES.rst

```diff
@@ -1,8 +1,116 @@
-.. _gammapy_1p0_release:
+.. _gammapy_0p16_release:
 
-1.0 (unreleased)
-----------------
-- Planned for Jan 2020
+0.16 (Jan 31, 2020)
+-------------------
+
+Summary
++++++++
+
+- Released Jan 31, 2020
+- 8 contrinutors
+- 60 pull requests (not all listed below)
+
+**What's new**
+
+For Gammapy v0.16 a ``FoVBackgroundMaker`` was implemented, which supports
+different methods of adapting the norm and tilt of a filed of view background
+model to the data.
+
+To provide a visual overview of the available models in Gammapy a
+`model gallery <modeling/gallery/index.rst>`__ was added. A general introduction
+on how to work with the different models is now avilable in a dedicated `models tutorial <notebooks/models.html>`__.
+The spectral analysis of an extended source is demonstrated in the newly
+added `extended source spectral analysis tutorial <notebooks/extended_source_spectral_analysis.ipynb>`__.
+
+The ``coordsys`` attribute of ``WcsGeom`` and ``HpxGeom`` was
+renamed to ``frame`` and now supports arbitrary Astropy coordinate
+frames.
+
+The ``Datasets`` and ``Models`` container objects now required unique
+names of the objects contained. By default unique identifier are generated
+in the model and dataset objects. The ``Datasets``, ``Models`` as well
+as ``Observations`` container classes, were extend to now support
+in place ``.append()``, ``.extend()`` and ``.insert()`` operations.
+
+For Gammapy v0.16 the API of the ``SensitivityEstimator`` and ``TSMapEstimator``
+was adapted to take a ``MapDataset`` or ``MapDatasetOnOff`` as input.
+The ``ASmooth`` class was renamed to ``ASmoothMapEstimator`` and also
+adapted to work with ``MapDataset`` and ``MapDatasetOnOff``.
+
+To further improve API consistency the ``EnergyDispersion`` class
```

Comment:
Why not start with this since this is the change most affecting users?

Reply:
I moved it up as the 3rd paragraph.